src/posix/Makefile-posix

```diff
@@ -94,7 +94,10 @@ configure_OPTIONS                   = \
 # Platform specific switches
 
 ifeq ($(DAEMON),1)
-configure_OPTIONS              += --enable-posix-daemon
+configure_OPTIONS              += \
+	--enable-posix-daemon 		  \
+	LIBS=-lstdc++				  \
```

Comment:
nit: alignment

Reply:
> nit: alignment

Excuse me, I failed to find the difference in you suggestion. Looks like the alignment is the same as my commit.
Can you elaborate your change?